prisma/schema.prisma

```diff
@@ -62,8 +61,7 @@ model User {
   firstName   String?
   lastName    String?
   birthdate   DateTime?
-  // horses      Horse[]
-
+  password    String? // temporary
```

Comment:
should be required? or is it not required because we're also supporting 3rd-party auth?

Reply:
It was because of 3rd party auth that I made it optional